ketcher-autotests/tests/File-Management/Open-And-Save-Files/Open-File/Load-Fragment/load-fragment.spec.ts

```diff
@@ -0,0 +1,205 @@
+import { test, expect } from '@playwright/test';
+import {
+  AtomButton,
+  clickInTheMiddleOfTheScreen,
+  pressButton,
+  takeEditorScreenshot,
+  waitForLoad,
+  openFileAndAddToCanvas,
+  openPasteFromClipboard,
+  waitForPageInit,
+  selectAtomInToolbar,
+  selectLeftPanelButton,
+  dragMouseTo,
+  LeftPanelButton,
+  selectDropdownTool,
+  getCoordinatesOfTheMiddleOfTheScreen,
+  moveMouseToTheMiddleOfTheScreen,
+  resetCurrentTool,
+  receiveFileComparisonData,
+  saveToFile,
+} from '@utils';
+import { getKet } from '@utils/formats';
+
+test.describe('load as fragment (Add to Canvas) srtuctures from files with different formats', () => {
+  test.beforeEach(async ({ page }) => {
+    await waitForPageInit(page);
+  });
+
+  test.afterEach(async ({ page }) => {
+    await takeEditorScreenshot(page);
+  });
+
+  test('Load Fragment - Load as fragment (Add to Canvas) - mol-files', async ({
+    page,
+  }) => {
+    /*
+     * Test case: EPMLSOPKET-1854
+     */
+    await openFileAndAddToCanvas('Molfiles-V2000/glutamine.mol', page);
+    await openFileAndAddToCanvas('Molfiles-V2000/cyclopentyl.mol', page);
+  });
+
+  test('Load Fragment - Load as fragment (Add to Canvas) - rxn-files', async ({
+    page,
+  }) => {
+    /*
+     * Test case: EPMLSOPKET-1860
+     */
+    await openFileAndAddToCanvas('Molfiles-V2000/cyclopentyl.mol', page);
+    // Coordinates for rectangle selection
+    const startX = 525;
+    const startY = 260;
+    const endX = 765;
+    const endY = 460;
+    await selectDropdownTool(page, 'select-rectangle', 'select-rectangle');
+    await page.mouse.move(startX, startY);
+    await page.mouse.down();
+    await page.mouse.move(endX, endY);
+    await page.mouse.up();
+
+    const xForMouseClick = 555;
+    const yForMouseClick = 420;
+    await page.mouse.move(xForMouseClick, yForMouseClick);
+
+    const xForMoveElement = 200;
+    const yForMoveElement = 180;
+    await dragMouseTo(xForMoveElement, yForMoveElement, page);
+
+    await openFileAndAddToCanvas('Rxn-V2000/reaction-3.rxn', page);
+  });
+
+  test('Load Fragment - Load as fragment (Add to Canvas) - smi-files', async ({
+    page,
+  }) => {
+    /*
+     * Test case: EPMLSOPKET-1861
+     */
+    await openFileAndAddToCanvas('Molfiles-V2000/glutamine.mol', page);
+    await openFileAndAddToCanvas('SMILES/1840-cyclopentyl.smi', page);
```

Comment:
Same question: Is it supposed to be here that one structure is placed on top of the second? Or is it worth specifying coordinates for the second fragment at a distance from the first structure?

Reply:
The same as previous: first structure is automatically moved before adding the second one.